src/Core/Domain/MailTemplate/CommandHandler/GenerateThemeMailTemplatesCommandHandler.php

```diff
@@ -108,25 +108,21 @@ public function handle(GenerateThemeMailTemplatesCommand $command)
     }
 
     /**
-     * When installing a new Language, if it's a new one the Translator component can't manage it because its cache is
-     * already filled with the default one as fallback. We force the component to update its cache by adding a fake
-     * resource for this locale (this is the only way clean its local cache)
+     * When installing a new language, we must add the lang pack to the translator
      *
-     * @param string $locale
+     * @param $locale
```

Comment:
Don't remove the parameter signature here!
```suggestion
     * @param string $locale
```